shared/utils/src/common.js

```diff
@@ -282,6 +282,7 @@ mclass[mclasssv] = {
 	key: mclasssv
 }
 
+//TODO：CNV_amp should be repalced with CNV_gain to avoid confusion with CNV amplification (CNV_amplification) below
```

Comment:
unfortunately "CNV_amp" may have to stay as-is since it may be hardcoded in lots of data beyond portal code.